UPGRADING

```diff
@@ -39,6 +39,17 @@ PHP 8.4 UPGRADE NOTES
     for invalid modes. Previously invalid modes would have been interpreted as
     PHP_ROUND_HALF_UP.
 
+- XML:
+  . The xml_set_*_handler() functions now declare and check for an effective
+    signature of callable|string|null for the $handler parameters.
+    Moreover, values of type string which correspond to method names,
```

Comment:
```suggestion
    Moreover, values of type string that correspond to method names,
```

Very silly nit: that is restrictive, which is descriptive. It's a pitfall I often fall into myself and I've gotten so many remarks on this in my own written texts that I notice it often now :sweat_smile:

Reply:
TIL